spec/factories.rb

```diff
@@ -0,0 +1,19 @@
+FactoryBot.define do
+  
+  factory :user do
+    name 'Test Name'
```

Comment:
although not required, if you want you can also take a look over [Faker](https://github.com/faker-ruby/faker)